includes/class-bsf-custom-fonts-admin.php

```diff
@@ -93,7 +95,7 @@ public function custom_fonts_menu_highlight() {
 			global $parent_file, $submenu_file;
```

Comment:
Why are we using $parent_file global variable?

Reply:
It is compulsory to override the global variable